YACReader/viewer.cpp

```diff
@@ -179,6 +185,7 @@ void Viewer::createConnections()
     connect(render, SIGNAL(numPages(unsigned int)), goToFlow, SLOT(setNumSlides(unsigned int)));
     connect(render, SIGNAL(numPages(unsigned int)), goToDialog, SLOT(setNumPages(unsigned int)));
     //connect(render,SIGNAL(numPages(unsigned int)),this,SLOT(updateInformation()));
+    connect(render, QOverload<unsigned int>::of(&Render::numPages), this, &Viewer::comicLoaded);
```

Comment:
`QOverload` was introduced in Qt 5.7, so YACReader will require Qt 5.7+ when this PR is merged. We can use more concise [`qOverload`](https://doc.qt.io/qt-5/qtglobal.html#qOverload) when we start requiring C++14 or C++17.

#197 disambiguates `Render::numPages` signal by renaming it to `Render::numPagesChanged`. Whichever of these two PRs is merged later, will have to be rebased and made compile.

Reply:
No longer a problem because Qt 5.15 is the minimum supported version now. Switched to `qOverload` as C++17 is now required.

#197 will still have to be rebased (it has other conflicts now too).